MaterialDesignThemes.Wpf/ResourceDictionaryExtensions.cs

```diff
@@ -32,8 +32,8 @@ public static void SetTheme(this ResourceDictionary resourceDictionary, ITheme t
 
             //NB: These are here for backwards compatibility, and will be removed in a future version.
             //These will be removed in version 4.0.0
-            SetSolidColorBrush(resourceDictionary, "SecondaryAccentBrush", theme.SecondaryMid.Color);
-            SetSolidColorBrush(resourceDictionary, "SecondaryAccentForegroundBrush", theme.SecondaryMid.ForegroundColor ?? theme.SecondaryMid.Color.ContrastingForegroundColor());
+            SetSolidColorBrush(resourceDictionary, "SecondaryHueMidBrush", theme.SecondaryMid.Color);
+            SetSolidColorBrush(resourceDictionary, "SecondaryHueMidForegroundBrush", theme.SecondaryMid.ForegroundColor ?? theme.SecondaryMid.Color.ContrastingForegroundColor());
 
```

Comment:
@Keboo These lines should be removed, not renamed, since these are already here.

https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/7fc12fed708b27c64bebc22c2ca5842263faa811/MaterialDesignThemes.Wpf/ResourceDictionaryExtensions.cs#L28-L29

Reply:
Yep I have a branch with that too address